core/src/repair_service.rs

```diff
@@ -57,14 +57,11 @@ pub const MAX_DUPLICATE_WAIT_MS: usize = 10_000;
 pub const REPAIR_MS: u64 = 100;
 pub const MAX_ORPHANS: usize = 5;
 
-pub enum RepairStrategy {
-    RepairRange(RepairSlotRange),
-    RepairAll {
-        bank_forks: Arc<RwLock<BankForks>>,
-        completed_slots_receiver: CompletedSlotsReceiver,
-        epoch_schedule: EpochSchedule,
-        duplicate_slots_reset_sender: DuplicateSlotsResetSender,
-    },
+pub struct RepairInfo {
+    pub bank_forks: Arc<RwLock<BankForks>>,
+    pub completed_slots_receiver: CompletedSlotsReceiver,
+    pub epoch_schedule: EpochSchedule,
+    pub duplicate_slots_reset_sender: DuplicateSlotsResetSender,
 }
```

Comment:
@carllin Removed and replaced with a single strategy

Reply:
Awesome, looks great, thanks!